modules/core/client/app/init.js

```diff
@@ -12,7 +12,23 @@ angular.module(ApplicationConfiguration.applicationModuleName).config(['$locatio
   }
 ]);
 
-angular.module(ApplicationConfiguration.applicationModuleName).run(function ($rootScope, $state, Authentication) {
+angular.module(ApplicationConfiguration.applicationModuleName).run(function ($rootScope, $state, $http, Authentication, CORE_CONST) {
+
+  //set token header
+  var auth_token = localStorage.getItem('auth_token');
```

Comment:
@kokokenada, @mleanos, I was looking into how we should store the token and Stormpath recommends to put it in cookies rather than localStorage:
https://stormpath.com/blog/where-to-store-your-jwts-cookies-vs-html5-web-storage/
>Stormpath recommends that you store your JWT in cookies for web applications, because of the additional security they provide, and the simplicity of protecting against CSRF with modern web frameworks. HTML5 Web Storage is vulnerable to XSS, has a larger attack surface area, and can impact all application users on a successful attack.